### PR TITLE
Track in-progress checks

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

File: qa/rpc-tests/test_framework/util.py [hunk 1]
```diff
@@ -247,17 +249,72 @@ def hex_str_to_bytes(hex_str):
...
+                    peerIdx = p["subver"]
+                    # print (peerIdx)
+                    if peerIdx in interestingNodes:
+                        connectedTo.append(peerIdx)
+                graph[nodeIdx] = connectedTo
+            print("Node connection graph: ", graph)
+            if not is_connected(graph):
+                print('sync_blocks: bitcoind nodes cannot sync because they are not all connected')
+                pdb.set_trace()
+                raise Exception('sync_blocks: bitcoind nodes cannot sync because they are not all connected')
```
Comment (ptschip) [inline qa/rpc-tests/test_framework/util.py]: do you really need this pdb.set_trace()...I think this is just going to cause a travis hang and timeout if it gets triggered and we'll never get to the exception?

File: qa/rpc-tests/ctor.py [hunk 2]
```diff
@@ -136,14 +137,15 @@ def run_test (self):
 
         connect_nodes_bi(self.nodes,0,1)
         sync_blocks(self.nodes[0:2])
-
+        print("Nodes 0,1 at block: %d " % self.nodes[2].getblockcount()) 
         # make n0 send coin to itself 4*3 times
```
Comment (ptschip) [inline qa/rpc-tests/ctor.py:141]: if you intend to keep these maybe should use logging.info() for the print's

File: qa/rpc-tests/test_framework/util.py (hunk 1 above)
Comment (gandrewstone) [reply qa/rpc-tests/test_framework/util.py]: thanks, I didn't pass back thru and clean this PR up.

**Review by ptschip — APPROVED** [review]
looks good now , given the test will pass

**Pull request merged**